<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewBookDialog</class>
 <widget class="QDialog" name="ViewBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр книги</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 14pt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="authorLabel"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Год издания:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yearLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Страниц:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pagesLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="descriptionLabel">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Экземпляров:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="copiesLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Состояние:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="conditionLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Доступно:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="availableLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>